mps_youtube/commands/play.py

```diff
@@ -92,7 +84,21 @@ def play(pre, choice, post=""):
                 streams.preload(g.model[chosen + 1], override=override)
 
         try:
-            play_range(songlist, shuffle, repeat, override)
+            if not config.PLAYER.get or not util.has_exefile(config.PLAYER.get):
+                g.message = "Player not configured! Enter %sset player <player_app> "\
+                    "%s to set a player" % (c.g, c.w)
+                return
+            else:
+                if config.PLAYER.get.endswith("mpv.exe"):
+                    mpv = Mpv()
+                    mpv.play_range(songlist, shuffle, repeat, override)
+                elif config.PLAYER.get.endswith("mplayer.exe"):
+                    mplayer = Mplayer()
+                    mplayer.play_range(songlist, shuffle, repeat, override)
+                else:
+                    g.message = "Player not supported! Enter %sset player <player_app> "\
```

Comment:
It's probably best just to have a generic fallback, although a dedicated `Vlc` class and such would be good to have in the long run, since I don't know what players people might be using and we want to try not to break them.

Mpd and catt for example are known to work (https://github.com/mps-youtube/mps-youtube/issues/168, https://github.com/mps-youtube/mps-youtube/issues/607), although not perfectly.